File: spec/php/ZlibWithHeader78Test.php
<?php
namespace Kaitai\Struct\Tests;

class ZlibWithHeader78Test extends TestCase {
    public function testZlibWithHeader78() {
        $r = ZlibWithHeader78::fromFile(self::SRC_DIR_PATH . "/zlib_with_header_78.bin");
        $this->assertEquals("a quick brown fox jumps over", $r->data());
    }
}

File: spec/go/zlib_with_header_78_test.go
package spec

import (
	"os"
	"testing"

	"github.com/kaitai-io/kaitai_struct_go_runtime/kaitai"
	"github.com/stretchr/testify/assert"

	. "test_formats"
)

func TestZlibWithHeader78(t *testing.T) {
	f, err := os.Open("../../src/zlib_with_header_78.bin")
	if err != nil {
		t.Fatal(err)
	}
	s := kaitai.NewStream(f)

	var r ZlibWithHeader78
	err = r.Read(s, &r, &r)
	if err != nil {
		t.Fatal(err)
	}

	assert.EqualValues(t, "a quick brown fox jumps over", string(r.Data))
}
